packages/formatters/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@stoplight/spectral-formatters",
+  "version": "1.0.0",
+  "homepage": "https://github.com/stoplightio/spectral",
+  "bugs": "https://github.com/stoplightio/spectral/issues",
+  "author": "Stoplight <support@stoplight.io>",
+  "engines": {
+    "node": "^12.20 || >=14.13"
+  },
+  "license": "Apache-2.0",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "files": [
+    "/dist"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/stoplightio/spectral.git"
+  },
+  "scripts": {
+    "release": "semantic-release -e semantic-release-monorepo",
+    "postbuild": "copyfiles -u 1 \"./src/html/*.html\" \"./dist/\""
```

Comment:
```suggestion
   "pretest": "yarn prebuild",
    "prelint": "yarn prebuild",
    "prebuild": "node scripts/bundle-html-templates.mjs",
```

Reply:
you may also need to copy over that script I added https://github.com/stoplightio/spectral/pull/2474/files#diff-c889100b2ca9f8c4d4ec6b7a1cf8514839031ca73fa701bc5bc8d92f61e5ac8f

Reply:
Updated.